devices/homeetec.js

```diff
@@ -0,0 +1,277 @@
+const fz = require('../converters/fromZigbee');
```

Comment:
Please move this into `tuya.js` in the standard format (so no things like `MODEL_CONFIGURATIONS`, `HANDLERS` or `getCurtainWithGangDefintion`). Since the format is different then the standard format, it makes it quite hard to understand what is happening here.

Reply:
Tried to separate the configuration from the logic, no problem,
Just a question about moving code to Tuya, since it's labeled as different vendor shouldn't it locate in different file?
Thanks